src/classes/RD_RecurringDonations_BATCH.cls

```diff
@@ -86,8 +86,7 @@ Schedulable {
     private Boolean isBatchInProgress {
         get {
             if (isBatchInProgress == null) {
-                //in this context, we have at least 1 Batch running(this itself).
-                isBatchInProgress = new UTIL_BatchJobService().getBatchesInProgress(batchClassName).size() > 1;
+                isBatchInProgress = new UTIL_BatchJobService().isAnotherBatchInProgress(batchClassName);
```

Comment:
This is not a result of changes in this PR, but I found it mildly confusing that in some places we use isBatchInProgress, where it seems we really mean isAnotherBatchInProgress.  It takes a non-zero amount of time for another developer to work through that, and I'm not sure it is justified.  Not a huge problem, but I think it might be more clear (and more accurate) if we just used "another" as part of the name everywhere.

Reply:
That is good feedback for the differentiation @npsp-reedestockton . So, basically we can rename `isBatchInProgress` variable to `isAnotherBatchInProgress` or something else? Any suggestion on the `isAnotherBatchInProgress()` name?

Reply:
That seems fine to me.

Reply:
Or we can rename the `isBatchInProgress` variable to `isConcurrentBatch`? And similarly, `isAnotherBatchInProgress()` into `isConcurrentBatch()`?

Reply:
Sure, that would be fine as well.  That's a good option.

Reply:
The change has been made. Can you please review again?